template/src/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
+import * as slices from './features';
```

Comment:
I don't like the `import *` here.  I'd rather explicitly do `import counterReducer from "features/counter/counterSlice"`.

Similarly, I'd rather not bother with trying to reuse the `slice.name` field.  Just say `counter: counterReducer`.